src/main/java/seedu/storemando/logic/commands/SortDescendingQuantityCommand.java

```diff
@@ -17,9 +18,9 @@
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        List<Item> lastShownList = model.getFilteredItemList();
-        if (lastShownList.size() == 0) {
-            throw new CommandException(MESSAGE_NO_ITEMS_TO_SORT);
+        List<Item> currentList = model.getFilteredItemList();
+        if (currentList.size() == 0) {
```

Comment:
Would it be better if the check can utilize the existing list method isEmpty() instead of using == 0? Just a suggestion to consider.

Reply:
Good suggestion!